roles/xdebug/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# XDebug Remote Debugging
+xdebug_remote_enable: 'false'
+xdebug_remote_connect_back: 'false'
```

Comment:
nitpick: These booleans could be `0` or `1` like the other boolean vars. Whichever you choose, could make consistent with corresponding vars in `group_vars/development/php.yml`. If you stick with `false`, could drop quotes.